proto/data_test.go

```diff
@@ -219,11 +219,11 @@ func TestPrevKey(t *testing.T) {
 }
 
 func TestKeyString(t *testing.T) {
-	if KeyMax.String() != "\xff..." {
+	if KeyMax.String() != "\\xff..." {
```

Comment:
Keys can be binary, so they generally need to be printed with `%q`. I think it's incorrect to escape the backslash here since `"\\xff"` is a valid key starting with a backslash.